common-custom-user-data-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id 'java-gradle-plugin'
+    id 'maven-publish'
+}
+
+group = "com.gradle"
+version = "0.1"
+
+repositories {
+    gradlePluginPortal()
+}
+
+dependencies {
+    compileOnly "com.gradle:gradle-enterprise-gradle-plugin:3.5.1"
```

Comment:
Please mention to @jonnybbb that the GE plugin release process needs to be adjusted to also bump up the version here.

Reply:
👍